doc/architecture/decisions/0003-generate-file-containing-organisation-data.md

```diff
@@ -0,0 +1,26 @@
+# 3. Store local copy of Organisation Data
+
+Date: 2017-06-12
+
+## Status
+
+Accepted
+
+## Context
+
+The [NHS Organisation API](http://api.nhs.uk/organisations) holds information about pharmacies.
+The [Connecting to Services](https://github.com/nhsuk/connecting-to-services) applications needs to make use of Pharmacy data.
+
+## Decision
+
+The NHS Organisation API should be scraped nightly to generate a file containing pharmacy data.
```

Comment:
Nygard article recommends active voice (e.g. The NHS Organisation API **will** be scraped nightly ).